src/utils.ts

```diff
@@ -106,6 +113,7 @@ export function loadScript(
 
   const queryString: URLSearchParams = new URLSearchParams({
     id,
+    l: config.dataLayerName ?? 'dataLayer',
```

Comment:
IMO we should add the `l` query param only when it is **not** equals `'dataLayer'`